docs/examples/example_orbit.rst

```diff
@@ -55,7 +55,7 @@ the pysat repository at ``demo/cnofs_vefi_dc_b_orbit_plots.py``.
        # pandas documentation for more information about the `resample` mtehod.
        # The 1-s cadence was chosen because it is the nominal cadence for this
        # instrument.
-       vefi.data = vefi.data.resample('1S', label='left').ffill(limit=1)
+       vefi.data = vefi.data.resample('1s', label='left').ffill(limit=1)
```

Comment:
Is this backwards compatible? On a different pull it looked like pandas doesn't have any overlap in the support for the keywords.

Reply:
Same comment for all of the frequency keyword changes.

Reply:
Currently passing tests in the operational branch for:
pandas-1.1.5

Reply:
The overlap issue had to do with the year-end and month-end keywords, which were new with 2.2.

Reply:
Both the new keywords for 's' and 'ns' work as expected in pandas 1.5.0.